mail-filter/spampd/spampd-2.42.ebuild

```diff
@@ -1,25 +1,25 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=8
 
 DESCRIPTION="spampd is a program to scan messages for Unsolicited Commercial E-mail content"
 HOMEPAGE="http://www.worlddesign.com/index.cfm/rd/mta/spampd.htm"
 SRC_URI="https://github.com/mpaperno/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc x86"
-IUSE=""
 
-RDEPEND="dev-lang/perl
+RDEPEND="
+	dev-lang/perl
```

Comment:
fwiw usually Perl doesn’t need to be in DEPEND unless building e.g XS modules and would need c
:= in RDEPEND then anyway

But possible the configure script is checking for it so it’s a BDEPEND?

Reply:
Hmm there isn't any sort of build system, it's just one perl script

Reply:
Just RDEPEND then